dlp/api/Snippets.Tests/SendDataToTheHybridJobTriggerTests.cs

```diff
@@ -0,0 +1,94 @@
+        private DlpTestFixture _fixture;
+        public SendDataToTheHybridJobTriggerTests(DlpTestFixture fixture) => _fixture = fixture;
+
+        [Fact]
+        public void TestHybridJob()
+        {
+            var dlp = DlpServiceClient.Create();
+
+            // Create the Job Trigger.
+            var request = new CreateJobTriggerRequest
+            {
+                ParentAsLocationName = new LocationName(_fixture.ProjectId, "global"),
+                JobTrigger = new JobTrigger
+                {
+                    Triggers =
+                    {
+                        new JobTrigger.Types.Trigger
+                        {
+                            Manual = new Manual()
+                        }
+                    },
+                    InspectJob = new InspectJobConfig
+                    {
+                        Actions =
+                        {
+                            new Action
+                            {
+                                JobNotificationEmails = new Action.Types.JobNotificationEmails(),
+                                PublishToStackdriver = new Action.Types.PublishToStackdriver(),
+                            }
+                        },
+                        InspectConfig = new InspectConfig
+                        {
+                            InfoTypes =
+                            {
+                                new InfoType[]
+                                {
+                                    new InfoType { Name = "EMAIL_ADDRESS" }
+                                }
+                            },
+                            MinLikelihood = Likelihood.Possible,
+                            IncludeQuote = true,
+                        },
+                        StorageConfig = new StorageConfig
+                        {
+                            HybridOptions = new HybridOptions
+                            {
+                                Description = "Hybrid job trigger for data from the comments field of a table that contains customer appointment bookings",
+                            },
+
+                        }
+                    }
+                }
+            };
+
+            var hybridJobTrigger = dlp.CreateJobTrigger(request);
+            var text = "My email is ariel@example.org and name is Ariel.";
+            var result = SendDataToTheHybridJobTrigger.SendToHybridJobTrigger(_fixture.ProjectId, hybridJobTrigger.JobTriggerName.JobTriggerId, hybridJobTrigger.Name, text);
+            Assert.Equal(DlpJob.Types.JobState.Active, result.State);
+            Assert.Contains(result.InspectDetails.Result.InfoTypeStats, f => f.InfoType.Name == "EMAIL_ADDRESS");
+
+            // Delete the created job trigger.
+            dlp.DeleteJobTrigger(new DeleteJobTriggerRequest
+            {
+                Name = hybridJobTrigger.Name
+            });
+
+            // Delete the activated job.
+            JobsDelete.DeleteJob(result.Name);
```

Comment:
This need to be on a finally on a try catch block. Else, if the test fails we won't delete the job.